tests/Dfc.CourseDirectory.WebV2.Tests/FeatureTests/EditVenue/SaveTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Dfc.CourseDirectory.Core.DataStore.CosmosDb.Queries;
+using Dfc.CourseDirectory.Testing;
+using Dfc.CourseDirectory.WebV2.Features.EditVenue;
+using FormFlow;
+using Microsoft.Extensions.DependencyInjection;
+using OneOf;
+using OneOf.Types;
+using Xunit;
+
+namespace Dfc.CourseDirectory.WebV2.Tests.FeatureTests.EditVenue
+{
+    public class SaveTests : MvcTestBase
+    {
+        public SaveTests(CourseDirectoryApplicationFactory factory)
+            : base(factory)
+        {
+        }
+
+        [Theory]
+        [InlineData(TestUserType.ProviderSuperUser)]
+        [InlineData(TestUserType.ProviderUser)]
+        public async Task Post_UserCannotAccessVenue_ReturnsForbidden(TestUserType userType)
+        {
+            // Arrange
+            var providerId = await TestData.CreateProvider(ukprn: 12345);
+            var venueId = await TestData.CreateVenue(providerId, email: "person@provider.com");
+
+            var anotherProviderId = await TestData.CreateProvider(ukprn: 67890);
+            
```

Comment:
Whitespace error here according to git show. Remove trailing whitespace. We should update editorconfig to trim trailing spaces.

Reply:
Fixed